### DISCUSSION TITLE
GitHub packages API - Internal Server Error when used with GITHUB_TOKEN

### DISCUSSION THREAD

**Discussion by ollosh** [community/community#38572] [Packages]
My initial problem was with deleting packages, but as I started digging a bit deeper I've noticed that API breaks when using GITHUB_TOKEN. 
Job described below results in 500 as it can be seen on the SS: 

```
- name: Get package
  run: |
    curl -H "Accept: application/vnd.github+json" \
      -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
      "https://api.github.com/orgs/${{ github.repository_owner }}/packages?package_type=container"
```
![image](https://user-images.githubusercontent.com/12979324/200577790-e58ed108-7d04-442d-bd97-6ff0472eab6c.png)

**Comment by ossanna16** [comment]
Hi there @ollosh and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
